compiler/AST/AggregateType.cpp

```diff
@@ -44,8 +44,10 @@ AggregateType* dtOnBundleRecord    = NULL;
 AggregateType* dtTaskBundleRecord  = NULL;
 AggregateType* dtError             = NULL;
 
-AggregateType::AggregateType(AggregateTag initTag)
-  : Type(E_AggregateType, NULL) {
+AggregateType::AggregateType(AggregateTag initTag) :
+  Type(E_AggregateType, NULL),
+
+  defaultTypeConstructor(NULL) {
```

Comment:
The other fields are initialized within the body of the constructor.  For consistency
this one should be too.